index.html

```diff
@@ -10,12 +10,168 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 
 
+<nav class="container-fluid bg-dark desktop">
+    <ul class="nav gap-4 container justify-content-end p-2">
+        <li class="nav-item"><i class="fa fa-facebook text-light"></i></li>
+        <li class="nav-item"><i class="fa fa-twitter text-light "></i></li>
+        <li class="nav-item text-light">English</li>
+        <li class="nav-item text-light">My page</li>
+        <li class="nav-item text-light">log out</li>
+    </ul>
+</nav>
```

Comment:
- [ ] As suggested by the previous reviewer,  I would kindly suggest you move this `nav` tag of your top navigation inside the body of your project in the `header` tag and it should be removed from the `head` section of your project  which is meant only to contain the meta tags, links and your app title👍 👍

Reply:
Hi @vickymarz ,
Sorry, I didn't understand, what should I make changes?